packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/AutolinkedNativeModules.g.cs

```diff
@@ -0,0 +1,13 @@
+﻿// AutolinkedNativeModules.g.cs contents generated by "react-native autolink-windows"
+
+using System.Collections.Generic;
+
+namespace Microsoft.ReactNative.Managed
+{
+    internal static class AutolinkedNativeModules
```

Comment:
For C# it could be done a bit cleaner by having a regular file with a partial method checked in and generate into the OBJ tree, and same with the .targets file for the referneces, to have that in the obj directory.

But I recognize that the convention for react native developers is to check in generated files... I still get a jolt that this is a big no-no. 

Maybe beceause the file is a `.g.cs`. Where it is more a `.lock.cs` since it only gets updated when you run the autolink tool. to me a .g.cs means it is regeenrated on every boyd...

Reply:
It will get re-generated on every build, that's how autolinking works.

I'm not going to change this any further, every request to change this means yet another separate breaking change PR I have to backport into 62.